src/preFilter.ts

```diff
@@ -18,14 +19,18 @@ export function filterByContent(searchee: Searchee): boolean {
 		});
 	}
 
-	const isSingleEpisodeTorrent =
-		searchee.files.length === 1 && EP_REGEX.test(searchee.files[0].name);
-
-	if (!includeEpisodes && isSingleEpisodeTorrent) {
+	const isSingleEpisodeTorrent = searchee.files.length === 1 && EP_REGEX.test(searchee.name); 
+	const isSeasonPack = typeof searchee.path !== 'undefined' ? SEASON_REGEX.test(path.dirname(searchee.path).split(path.sep).reverse()[0]): false;
+	
+	if (!includeEpisodes && isSingleEpisodeTorrent && !isSeasonPack) {
 		logReason("it is a single episode");
 		return false;
 	}
 
+	if (!includeSeasonPackEpisodes && (searchee.files.length === 1 && isSeasonPack)) {
```

Comment:
out of curiosity, does isSingleEpisodeTorrent not work here?
```suggestion
	if (!includeSeasonPackEpisodes && isSingleEpisodeTorrent && comesFromSeasonPack) {
```

Reply:
I think my logic at the time was based on certain season pack types with very poor or absolute naming schemes, specifically anime (absolute numbering which would not match EP_REGEX) and daily or soap opera shows.

It occurred to me that this was incredibly edge-case outside of anime but with a comparison as simple a number...I just opted for this instead.

Reply:
good point about the anime, I like that. Let's move the files.length === 1 check into the isSeasonPack variable above though

Reply:
and then probably the best name for the variable after that change is "isSeasonPackEpisode"

Reply:
Sounds perfect.